feature/amazon/src/main/java/com/revenuecat/purchases/amazon/handler/PurchaseHandler.kt

```diff
@@ -96,7 +92,16 @@ class PurchaseHandler : PurchaseResponseListener {
     }
 
     private fun onFailed(onError: (PurchasesError) -> Unit) {
-        // Indicates that the purchase failed. Can simply mean user cancelled
+        // Indicates that the purchase failed. Can simply mean user cancelled.
+        //
+        // According to Amazon's EU MFA flow documentation, the flow would
+        // return a PurchaseResponse.RequestStatus.FAILED if cancelled and a
+        // PurchaseResponse.RequestStatus.SUCCESSFUL if the MFA flow is completed successfuly
+        //
+        // Amazon docs:
+        // https://developer.amazon.com/blogs/appstore/post/
+        // 72ee4001-9e54-49b1-83ad-f1cede9b40ce/
+        // ensuring-your-implementation-of-in-app-purchase-is-ready-for-eu-multi-factor-authentication
```

Comment:
let's not forget to swap this with a rev.cat before shipping the final version, not needed for this PR though

Reply:
I will add a TODO

Reply:
I added another TODO here 7e500daed42ee5726d73e5bd646e769b6e5dac77

I will clean up all these TODOs before merging to develop